spec/factories/journal_entries.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
```

Comment:
don't we need factories for other models?

Reply:
yes your right later i will need them but so far i haven't added any functionality for the other views so i will ad the factories when i add tests for the other controllers once their used